ci/Dockerfile.circle_build

```diff
@@ -0,0 +1,41 @@
+# To build and deploy, from this directory:
```

Comment:
Possible this could be omitted for now, and we continue using the workbench image. The circle config above still uses the older one

Reply:
I believe the docker image above is what we've published to dockerhub? But that's just the base image? If I am wrong, let me know! I can fix.

Reply:
Sorry, by above I was actually referring to the circle config file "- image: allofustest/workbench:buildimage-0.0.12". There's no reason to have this Dockerfile in multiple locations if they have the same exact image name. One will just overwrite the other when you publish the docker image.

Reply:
rm file?

Reply:
Should be removed now. I was missing what the file was actually doing. (I thought it was loading in that docker image, not designed to publish it)